clientv3/op.go

```diff
@@ -225,3 +228,11 @@ func WithLastRev() []OpOption { return withTop(SortByModifiedRev, SortDescend) }
 func withTop(target SortTarget, order SortOrder) []OpOption {
 	return []OpOption{WithPrefix(), WithSort(target, order), WithLimit(1)}
 }
+
+// WithProgressNotify makes watch server send periodic status updates.
+// The default interval is 10-minute.
```

Comment:
remove the default interval thing. this is not configurable. server decides when to send a progress notification.